src/routers/datatypeRouters/edges/variants_genes.ts

```diff
@@ -122,6 +110,48 @@ async function qtlSearch (input: paramsFormatType): Promise<any[]> {
 
   return objects
 }
+async function getQTLedges(input: paramsFormatType, customFilters: string = ''): Promise<any[]> {
+  let limit = QUERY_LIMIT
+  if (input.limit !== undefined) {
+    limit = (input.limit as number <= MAX_PAGE_SIZE) ? input.limit as number : MAX_PAGE_SIZE
+    delete input.limit
+  }
+
+  let filterBy = ''
+  const filterSts = getFilterStatements(qtls, input)
+  if (filterSts !== '') {
+    filterBy = `FILTER ${filterSts}`
+  }
+
+  if (customFilters !== '') {
+    customFilters = ` AND ${customFilters}`
+  }
+
+  const sourceQuery = `FOR otherRecord IN variants
+  FILTER otherRecord._id == record._from
+  RETURN {${getDBReturnStatements(variantSchema).replaceAll('record', 'otherRecord')}}
+  `
+
+  const targetQuery = `FOR otherRecord IN genes
+  FILTER otherRecord._id == record._to
+  RETURN {${getDBReturnStatements(geneSchema).replaceAll('record', 'otherRecord')}}
+  `
+
+  const query = `
+    FOR record IN variants_genes
+    ${filterBy} ${customFilters}
```

Comment:
is there a case where we have `customFilters` but no `filterBy`?

Reply:
I think `customFilters` are always added when there's filter on effect_size/log10pvalue, which are stored in `filterBy`, so shouldn't be such cases?